documentation/getting_started.md

```diff
@@ -645,13 +676,14 @@ successfully installed and launched on the device.
 ### Run Tizen-to-Android Offloading
 
 To run Tizen-to-Android Offloading you need to connect both devices to the same Wi-Fi network or
-ethernet. Press the start button on the Android device to run the server. Then, the port number the
-server is listening will display on the screen like the following screenshot.
+ethernet. Press the load models button on the Android device to run the offloading service
+based on the conf files. Then, the port number to which the service is listening will display
+on the screen like the following screenshot.
 
 ![Android Device (Start the server): A-34-arm64](img/A-34-arm64_port.jpg)
 
-Use android's port number in Tizen application and create pipelines by pressing buttons. When the
-image classification task runs successfully, the detected label and inference time are displayed on
+Tizen devices automatically find offloading services using the model ID. When the image
+classification task runs successfully, the detected label and inference time are displayed on
 the screen as shown below.
 
 ![Tizen Device (Start the client): T-8.0-armv7l](img/T-8.0-armv7l_run.png)
```

Comment:
This comment is not directly related with this change, but it seems that there are too many meaningless decimal points for inference time in picture.
IMO, three decimal points are enough.

Reply:
I agree
https://github.com/nnstreamer/nnstreamer-example/pull/360
Added this PR and updates screenshot

Thank you :)